org.lflang.tests/src/org/lflang/tests/compiler/LetInferenceTests.java

```diff
@@ -0,0 +1,137 @@
+package org.lflang.tests.compiler;/* Parsing unit tests. */
+
+/*************
+ Copyright (c) 2019, The University of California at Berkeley.
```

Comment:
The date might need to be update. Also, I think you might want to add Hanyang University as well.

Reply:
I reflected on your comment in https://github.com/lf-lang/lingua-franca/pull/1292/commits/ed0e00e44a54d108d45a0b269986c36333481ba2.